### PR TITLE
Back 335 - Short Cache for Balance calculation 

### PR DESCRIPTION
**Pull request by RomanWlm**
Add Caching for Balance 
- Configurable size : balance.cache.max_size -> Default 1000
- Configurable TTL : balance.cache.ttl_min -> Default 1

### CODE REVIEW

**Comment by luchenyuxx** [comment]
pushed the fix of fallback http client.

File: src/main/scala/co/ledger/wallet/daemon/models/Account.scala [hunk 1]
```diff
@@ -7,7 +7,7 @@ import cats.instances.list._
 import cats.syntax.either._
 import cats.syntax.traverse._
 import co.ledger.core
-import co.ledger.core._
+import co.ledger.core.{BigInt, _}
```
Comment (aymeric-ledger) [inline src/main/scala/co/ledger/wallet/daemon/models/Account.scala]: ?

File: src/main/scala/co/ledger/wallet/daemon/clients/ApiClient.scala [hunk 2]
```diff
@@ -109,22 +109,29 @@ class ApiClient(implicit val ec: ExecutionContext) extends Logging {
...
+                case Some(proxy) => client.withTransport.httpProxyTo(s"${host}:443").withTls(host).newService(s"${proxy.host}:${proxy.port}")
+                case None => client.withTls(host).newService(s"${host}:443")
+              }
+              Some(FallbackParams(s"${host}:443", "/" + query), c)
+            case _ =>
+              None
+          }
+        } yield r
+      }.collect{
+      case (currency, opt) if opt.isDefined => (currency, opt.get)
```
Comment (aymeric-ledger) [inline src/main/scala/co/ledger/wallet/daemon/clients/ApiClient.scala]: Indentation please.

**Pull request merged**